crates/tabby/src/services/health.rs

```diff
@@ -26,19 +33,21 @@ impl HealthState {
     pub fn new(model: Option<&str>, chat_model: Option<&str>, device: &Device) -> Self {
         let (cpu_info, cpu_count) = read_cpu_info();
 
-        let cuda_devices = match read_cuda_devices() {
+        let gpu_devices = match read_gpu_devices() {
             Ok(s) => s,
             Err(_) => vec![],
         };
 
+        #[allow(deprecated)]
         Self {
             model: model.map(|x| x.to_owned()),
             chat_model: chat_model.map(|x| x.to_owned()),
             device: device.to_string(),
             arch: ARCH.to_string(),
             cpu_info,
             cpu_count,
-            cuda_devices,
+            gpu_devices: gpu_devices.clone(),
+            cuda_devices: gpu_devices,
```

Comment:
Since you added a new field `gpu_device`, let's keep cuda_devices' original semantic (only include nvidia cuda devices).

Reply:
Sure can do that if you really want... originally it was also displaying cuda devices though that weren't being used. So my thought is that applications not updated to properly check for non-CUDA devices yet, will just display them as cuda devices for now, which is technically wrong, but at least they are displaying them and don't just silently drop them. I think that would be better for users of non-NVIDIA GPUs than the software displaying nothing at all, but if you see that differently, I can change that.

Reply:
Fixed